boards/arm/nrf9160_pca10090/nrf9160_pca10090.yaml

```diff
@@ -0,0 +1,9 @@
+identifier: nrf9160_pca10090
+name: nRF9160-PCA10090
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+ram: 64
+flash: 256
```

Comment:
Seems like we should have some list of 'supported' devices here.

Reply:
I can try to add a list here (not consistent among ARM or even NRF boards, though)